TELCODOCS-1481 CNF-9121 Reduce DU Profile Apply Time (#68392)
openshift/openshift-docs: modules/ztp-deploying-a-site.adoc

For more information about BMC addressing, see the "Additional resources" section.
====

.. Optional: For {sno}, you can reduce installation time by applying Day 2 manifests at an earlier stage. 

> You do this using the `spec.clusters.clusterLabels.ztp-accelerated-provisioning label`, as in the following example:

Author: makes sense. Thanks

+
[source,yaml]
----
clusterLabels:
   common: true
   group-du-sno: ""
   sites : "example-sno"
   ztp-accelerated-provisioning: full
----
	

> .. You can inspect the default set of extra-manifest `MachineConfig` CRs in `out/argocd/extra-manifest`. It is automatically applied to the cluster when it is installed.

Reviewer: Out of scope and not necessary at all unless you're feeling very motivated, but this line can also be modified to use `Optional:` 
```suggestion
.. Optional: Inspect the default set of extra-manifest `MachineConfig` CRs in `out/argocd/extra-manifest`. It is automatically applied to the cluster when it is installed.
```
